core/src/components/input/input.tsx

```diff
@@ -363,6 +376,19 @@ export class Input implements ComponentInterface {
     forceUpdate(this);
   }
 
+  /**
+   * This prevents the native input from emitting the click event.
+   * Instead, the click event from the ion-input is emitted.
+   */
+  @Listen('click', { capture: true })
+  onClickCapture(ev: Event) {
+    const nativeInput = this.nativeInput;
+    if (nativeInput && ev.target === nativeInput) {
+      ev.stopPropagation();
+      this.el.click();
```

Comment:
Do we need to forward `ev` to this call or does it matter? I'm not sure if this directly triggers `onclick` from the element (in which case we should) or if it triggers other onclick code that then emits the event with its own event parameter passed.

Reply:
I checked the events that get emitted based on clicking on different parts of item and they are identical so it seems like this is fine since they are both triggered by programatic clicks.